README.md

```diff
@@ -254,6 +254,64 @@ helm template deploy/helm/eunomia-operator/ --set openshift.route.enabled=true |
 
 We've created several examples for you to test out Eunomia. See [EXAMPLES](examples/README.md) for details.
 
+## Monitoring with Prometheus
+
+[Prometheus](https://prometheus.io/) is an open-source systems monitoring and alerting toolkit.
+
+By default, the metrics are served on `0.0.0.0:8383/metrics`. To modify the port the metrics are exposed on, change the `var metricsPort int32 = 8383` variable in the `cmd/manager/main.go` file of the generated operator.
+
+### Usage:
+
+```go
+    import(
+        "github.com/operator-framework/operator-sdk/pkg/metrics"
+        "sigs.k8s.io/controller-runtime/pkg/manager"
+    )
+
+    func main() {
+
+        ...
+
+        // Change the below variables to serve metrics on different host or port.
+        var metricsHost = "0.0.0.0"
+        var metricsPort int32 = 8383
+
+        // Pass metrics address to controller-runtime manager
+        mgr, err := manager.New(cfg, manager.Options{
+            Namespace:          namespace,
+            MetricsBindAddress: fmt.Sprintf("%s:%d", metricsHost, metricsPort),
+        })
+
+        ...
+
+        // Create Service object to expose the metrics port.
+        _, err = metrics.ExposeMetricsPort(ctx, metricsPort)
+        if err != nil {
+            // handle error
+            log.Info(err.Error())
+        }
+
+        ...
+    }
+```
+
+[scrape_config](https://prometheus.io/docs/prometheus/latest/configuration/configuration/#scrape_config)
+[prometheus-kubernetes](https://github.com/prometheus/prometheus/blob/master/documentation/examples/prometheus-kubernetes.yml)
```

Comment:
I'm not sure if these are needed since we have the general Prometheus link above.
If we want to keep these, I'd say they should be included in statements explaining the need for them rather than just random links. Maybe something like:
```suggestion
For more details on scrape configurations, see the documentation on [prometheus.io](https://prometheus.io/docs/prometheus/latest/configuration/configuration/#scrape_config).
You can find additional examples on their [GitHub page](https://github.com/prometheus/prometheus/blob/master/documentation/examples/prometheus-kubernetes.yml).
```

Reply:
Done